woocommerce/woocommerce-ios
# [Shipping labels] Update collapsed bottom sheet UI in landscape

Closes: #14472


## Description


- Added the "Mark order complete" toggle next to the button (same as the expanded bottom sheet layout)
- Updated the toggle copy to align with web: "After purchasing a label, mark this order as complete and notify the customer"
- While looking at Android PR https://github.com/woocommerce/woocommerce-android/pull/12939 I noticed that even on portrait the "purchase" label button is visible, is that ok? I changed it here also to be visible. @rachelmcr 

## Testing information


- Install and set up the Woo Shipping extension on your store.
- Build and run the app with the revampedShippingLabelCreation feature flag enabled.
- Create an order with the processing status and at least one physical product.
- In your WooCommerce product settings on the web, change the weight and dimension units and save your changes.
 On the Orders tab in the app, open the order you created.
- In the order details, select "Create Shipping Label."
- Tap "Select a Package."
- Switch to Carriers or Saved tab and select one of the packages (optional step)
- Check the bottom sheet layout and content in both portrait and landscape (on iPhone), compare to screenshots below and check that the proper UI elements are shown.
- Check that the toggle copy is updated

## Screenshots


## Unselected

| Before     | After |
| ----------- | ----------- |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 42 41](https://github.com/user-attachments/assets/463a65ac-cca1-4dc5-8b3a-19a96a85d192) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 34 17](https://github.com/user-attachments/assets/c7c8480f-b32e-467e-964c-6614c3b7ecd2) |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 42 45](https://github.com/user-attachments/assets/6da8c019-fa1b-499b-afa7-beba97bc3c49) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 34 19](https://github.com/user-attachments/assets/e1dec147-1b07-4fa8-a6af-e0b01b8b086f) |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 42 47](https://github.com/user-attachments/assets/017a8a37-e503-40f4-b8ac-98cd750733de) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 34 22](https://github.com/user-attachments/assets/935d75c9-1067-4abd-8b5f-903462ea9f6f) |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 42 50](https://github.com/user-attachments/assets/f7c02cd5-6a0d-4088-aa95-105750481e63) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-18 at 10 12 11](https://github.com/user-attachments/assets/3e9de7ac-e72f-41f0-bd9c-bc16b673b964) |

## Selected package

| Before     | After |
| ----------- | ----------- |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 43 04](https://github.com/user-attachments/assets/eda7674e-9779-4e62-8893-69a25d8efa7c) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 34 34](https://github.com/user-attachments/assets/8c0d5dc7-5281-4718-bdf7-c40385dfbbe3) |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 43 06](https://github.com/user-attachments/assets/b2c9b8d6-ac9e-4a92-af71-a09da06955b0) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 34 36](https://github.com/user-attachments/assets/674879dc-835b-43e7-9d84-29669105af53) |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 43 11](https://github.com/user-attachments/assets/86275ddd-3031-4b2d-b759-bd7d128754ed) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 34 39](https://github.com/user-attachments/assets/cb0bf46b-957c-4db5-ba54-1073e6a3c29a) |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 43 14](https://github.com/user-attachments/assets/d07c7e49-86c2-4a82-b339-b85f4bc9f543) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-17 at 11 34 41](https://github.com/user-attachments/assets/58616059-626f-48d6-a0b1-c4f9ad510952) |

---


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
